test/ocsp_test.go

```diff
@@ -2438,3 +2438,17 @@ func getOCSPStatus(s tls.ConnectionState) (*ocsp.Response, error) {
 	}
 	return resp, nil
 }
+
+func TestOCSPManualConfig(t *testing.T) {
```

Comment:
Isn't this testing an implicit config, not a manual config?

Reply:
Actually renamed to `TestOCSPTLSConfigNoLeafSet`